src/ElectricPokemon.java

```diff
@@ -0,0 +1,124 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private static final String type = "electric";
+
+    private static final List<String> attacks = Arrays.asList("Thunderpunch", "Electroball", "Thunder", "Volttackle");
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+    public void thunderPunch(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " " + "hits" + " " + enemy.getName() + " " + "with a Thunderpunch!");
+        switch (enemy.getType()) {
+            case "grass": {
+                System.out.println(enemy.getName() + " " + "loses 20hp");
+                enemy.setHp(enemy.getHp() - 20);
+            }
+            break;
+            case "water": {
+                System.out.println(enemy.getName() + " " + "loses 40hp");
+                enemy.setHp(enemy.getHp() - 40);
+            }
+            break;
+            case "fire": {
+                System.out.println(enemy.getName() + " " + "loses 15hp");
+                enemy.setHp(enemy.getHp() - 15);
+            }
+            break;
+            case "electric": {
+                System.out.println(enemy.getName() + " " + "loses 10hp");
+                enemy.setHp(enemy.getHp() - 10);
+            }
+        }
+        System.out.println(enemy.getName() + " " + "has" + " " + enemy.getHp() + " " + "hp left");
+    }
+
+    public void electroBall(Pokemon name, Pokemon enemy) {
+        System.out.println(name.getName() + " " + "throws an Electroball and hits" + " " + enemy.getName());
+        switch (enemy.getType()) {
```

Comment:
Je switch statements zijn overal bijna hetzelfde. Het zou mooi zijn als je dit op een centrale plek zou kunne ndefinieren in een helper methode, zodat je het kunt hergebruiken.